azure/scope/managedcontrolplane.go

```diff
@@ -591,9 +591,11 @@ func (s *ManagedControlPlaneScope) ManagedClusterSpec() azure.ASOResourceSpecGet
 	if s.ControlPlane.Spec.AADProfile != nil {
 		managedClusterSpec.AADProfile = &managedclusters.AADProfile{
 			Managed:             s.ControlPlane.Spec.AADProfile.Managed,
-			EnableAzureRBAC:     s.ControlPlane.Spec.AADProfile.Managed,
```

Comment:
What's the default if `EnableAzureRBAC` doesn't get set explicitly in this new code path? I wonder if this could introduce some backward-compatibility issue.

Reply:
I think you're right! Since the user was able to fix their issue with ASOAPI, I'll go ahead and close this PR.

/close